build_projects/update-dependencies/Program.cs

```diff
@@ -60,6 +60,20 @@ public static void Main(string[] args)
             }
         }
 
+        private static BuildInfo GetBuildInfo(string name, string buildInfoFragment, bool fetchLatestReleaseFile = true)
```

Comment:
Moving this logic there makes sense to me. I didn't know we were adopting this convention at the time, and I didn't want to get in the business of determining whether a string is a URL or a path--but the `file://` scheme makes it easy.